modules/themoneytizerBidAdapter.md

```diff
@@ -0,0 +1,36 @@
+# Overview
+
+```
+Module Name: The Moneytizer Bid Adapter
+Module Type: Bidder Adapter
+Maintainer: tech@themoneytizer.com
+```
+
+## Description
+
+Module that connects to The Moneytizer demand sources
+
+## Test parameters
+
+```js
+
+var adUnits = [
+  {
+    code: 'your-adunit-code',
+    mediaTypes: {
+      banner: {
+        sizes: [[300, 250]],
+      },
+    },
+    bids: [
+      {
+        bidder: "themoneytizer",
+        params: {
+          pid: -1,
+          test: 1
+        },
```

Comment:
update with params that return valid bid responses from your endpoint, including user sync data in `body.c_sync`

Reply:
Only `pid` and `test` params return valid bid responses. `baseUrl` is unused currently and will be custom for specific publishers. Should I remove it for now?

user sync data added in `body.c_sync` ✅

Reply:
No problem to keep baseUrl.

For the test params, I am seeing an error returned:
```
Warning: Undefined property: stdClass::$schain in /var/www/html/m/class/client/parserClient.class.php on line 61
{"ads":"no-ads","err-msg":"E9"}
```

Reply:
should be fixed now